Packages/vcs/Lib/vcsvtk/isolinepipeline.py

```diff
@@ -27,6 +27,11 @@ def _updateVTKDataSet(self):
         data = vcs2vtk.numpy_to_vtk_wrapper(self._data1.filled(0.).flat,
                                             deep=False)
         self._vtkDataSet.GetPointData().SetScalars(data)
+        if (vcs2vtk._DEBUG_VTK):
+            writer = vtk.vtkXMLDataSetWriter()
+            writer.SetFileName("isoline-p.vts")
```

Comment:
I don't have a problem with that. We just have to make sure _DEBUG_VTK is not True otherwise a bunch of files are going to be created.  I am open to making this better, though. Production code = many bugs we know about and probably many more we don't know about. :-) So we should make the code as easier/faster to debug as possible. This kind of hooks are very useful and to me it does not make sense to remove them every time I commit something.